### DISCUSSION TITLE
Ability to differentiate Sponsors' Issue tickets.

### DISCUSSION THREAD

**Discussion by LlmDl** [community/community#4379] [Other Feature Feedback, Questions, & Ideas]
I'd love it if my Sponsors' issue tickets could stand out from the rest. Tickets made by them could be flaired or their name could appear a different colour, on the Issues page.

If you wanted to go above and beyond, if a sponsor were to comment on a ticket, that could also alter the ticket's appearance from the Issues page.

At the moment you have to hover over their name, which shows if they are a sponsor.

**Comment by bdresser** [comment]
@LlmDl this is a great idea! We've seen other users build similar functionality with actions and labels – [JasonEtco/is-sponsor-label-action](https://github.com/JasonEtco/is-sponsor-label-action) is one good example. I agree we can do more here in the product, so stay tuned :) 

We've also had some conversations about introducing a feature that lets you limit the ability to create new issues to your sponsors only. Is that something you'd find useful?

**Reply by LlmDl** [reply]
For my uses that would be a bit too far (not enough sponsors ha!) and I wouldn't get the bug reports I need. But I'm sure some people would want that.
